Automattic/wp-calypso
# Add storybook for comparison grid

Related to #

## Proposed Changes

This change adds the comparison grid to storybook and restructures the data mocks to make it more manageable.

<img width="1881" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/7bc1eff1-5336-4fed-b3ef-18308727717e">


## Testing Instructions



* Run yarn workspace @automattic/plans-grid-next storybook
* Go to the specified browser window
* The storybook instance should show the comparison grid as shown above

## Pre-merge Checklist
